packages/build/src/core/flags.js

```diff
@@ -91,9 +91,9 @@ Default: automatically guessed`,
     hidden: true,
   },
   telemetry: {
-    boolean: true,
+    boolean: false,
```

Comment:
The flag is still a boolean, so this should remain `true`?

Reply:
🤦 you're right, was on a rampage find and replace 😂 fixed in https://github.com/netlify/build/pull/2349/commits/c8434a49e47d9e8c73e9d2425f6fb0c3a79ebe27